From 5190e4b9b528b96c21f44af26558f87a42b3e3ad Mon Sep 17 00:00:00 2001
From: Richard Liu <xuegang.liu@nxp.com>
Date: Wed, 31 Oct 2018 21:39:14 +0800
Subject: [PATCH 5109/5242] MA-12928-2 [#imx-1199] Use gcdANDROID MARCO to
 control android build

commit  0a64d9dccb55fa24adea1d2e9f32ab5418edf985 from
https://source.codeaurora.org/external/imx/linux-imx.git

Should use the correct build MARCO gcdANDROID to control android build.

Change-Id: I56a745b228a24ba502dc9771596aad78c1d8d808
Signed-off-by: Richard Liu <xuegang.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/gpu-viv/hal/kernel/gc_hal_kernel.c     |    2 +-
 .../hal/kernel/gc_hal_kernel_video_memory.c        |    2 +-
 .../mxc/gpu-viv/hal/kernel/inc/gc_hal_metadata.h   |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/mxc/gpu-viv/hal/kernel/gc_hal_kernel.c b/drivers/mxc/gpu-viv/hal/kernel/gc_hal_kernel.c
index 97f0128..13db5d5 100644
--- a/drivers/mxc/gpu-viv/hal/kernel/gc_hal_kernel.c
+++ b/drivers/mxc/gpu-viv/hal/kernel/gc_hal_kernel.c
@@ -1690,7 +1690,7 @@
     }
     else
     {
-#ifdef ANDROID
+#ifdef gcdANDROID
         if (nodeObj->metadata.ts_address == 0 && nodeObj->tsNode != NULL)
         {
             gctUINT32 Address = 0;
diff --git a/drivers/mxc/gpu-viv/hal/kernel/gc_hal_kernel_video_memory.c b/drivers/mxc/gpu-viv/hal/kernel/gc_hal_kernel_video_memory.c
index f4ec7b4..a0d9ac62 100644
--- a/drivers/mxc/gpu-viv/hal/kernel/gc_hal_kernel_video_memory.c
+++ b/drivers/mxc/gpu-viv/hal/kernel/gc_hal_kernel_video_memory.c
@@ -2619,7 +2619,7 @@
 
     node->metadata.magic = VIV_VIDMEM_METADATA_MAGIC;
     node->metadata.ts_fd = -1;
-#ifdef ANDROID
+#ifdef gcdANDROID
     node->metadata.ts_address = 0;
 #endif
 
diff --git a/drivers/mxc/gpu-viv/hal/kernel/inc/gc_hal_metadata.h b/drivers/mxc/gpu-viv/hal/kernel/inc/gc_hal_metadata.h
index eae2c17..619c7a5 100644
--- a/drivers/mxc/gpu-viv/hal/kernel/inc/gc_hal_metadata.h
+++ b/drivers/mxc/gpu-viv/hal/kernel/inc/gc_hal_metadata.h
@@ -100,7 +100,7 @@
 
     int32_t  ts_fd;
     void *   ts_dma_buf;
-#ifdef ANDROID
+#ifdef gcdANDROID
     dma_addr_t ts_address;
 #endif
 
-- 
1.7.9.5

